bindings/cs/Protean-cs/Protean.Net/VariantCollection.cs

```diff
@@ -304,6 +321,7 @@ public IEnumerator<VariantItem> GetEnumerator()
 
         public int CompareTo(IVariantData rhs)
         {
+            // TODO - comparison of bags should not rely on element ordering
```

Comment:
Note that the other languages behave the same way AFAICS. Arguably it's not worth leaving this note, as it's not clear that we could change it. For the same reason, I wonder if it would be better to make `GetHashCode()` consistent with this behaviour.

Reply:
What's the point of having a bag then? Surely that use case would be covered by a list.